<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelMap">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>746</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>labelMap</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widgetAddTourist" native="true">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>20</y>
      <width>232</width>
      <height>137</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelAddTourist">
       <property name="text">
        <string>添加旅客：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>出发点：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxTouristStart"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>目的地：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxTouristDes"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>时限(小时):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditTouristLimit"/>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButtonAddTourist">
       <property name="text">
        <string>添加旅客</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widgetAddTourist_2" native="true">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>240</y>
      <width>232</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelAddTourist_2">
       <property name="text">
        <string>添加城市：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>城市名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditCityName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>风险值：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditCityRisk"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButtonAddCity">
       <property name="text">
        <string>添加城市</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widgetAddTourist_3" native="true">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>380</y>
      <width>235</width>
      <height>189</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="labelAddTourist_3">
       <property name="text">
        <string>添加时刻表：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>始发站：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxTransportStart"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>始发站：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxTransportDes"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>风险值：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>出发时间:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEditTransportStartTime"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>时长(小时):</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEditTransportTimecost"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonAddTransport">
       <property name="text">
        <string>添加时刻表</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEditTransportRisk"/>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidgetTourist">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>171</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidgetCity">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>171</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
